src/decisionengine/framework/tests/test_sample_config.py

```diff
@@ -15,6 +15,8 @@
     conf_path=TEST_CONFIG_PATH, channel_conf_path=TEST_CHANNEL_CONFIG_PATH
 )  # pylint: disable=invalid-name
 
+_STOPPED_CHANNEL_OPTS = {'Channel test_channel stopped cleanly.', 'Channel test_channel has been killed.'}
```

Comment:
The one unit test is reporting 

`Channel test_channel has been killed due to shutdown timeout (1 second).`

which I suspect needs to be added to this set....

Any reason you went with a set instead of a `list` or `tuple`?

Reply:
Oops, will fix.

I chose a set due to my misunderstanding of what container types are supported for the expression `x in s`, which is used in the assertions.  Will switch to a `list`.